service/resource/vaultpki/create.go

```diff
@@ -8,41 +8,12 @@ import (
 	"github.com/giantswarm/cert-operator/service/key"
 )
 
```

Comment:
* renamed `Process*State` -> `Apply*Change` and moved up
* renamed `Get*State` -> `new*Change` and moved down